HCP Terraform Docs (#2874)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/api-docs/stacks/deployments.mdx

| `page[number]` | **Optional.** If omitted, the endpoint will return the first page.                      |
| `page[size]`   | **Optional.** If omitted, the endpoint will return 20 Stack deployment groups per page. |

### Available Related Resources

> This GET endpoint can optionally return related resources, if requested with [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources). The following resource types are available:

Reviewer: ```suggestion
To retrieve related resources, you can optionally add [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources) when making `GET` requests. The following resource types are available:
```

|------------------------------|------------------------------------------------|
| `:stack_deployment_group_id` | The id of the Stack deployment group to fetch. |

### Available Related Resources

> This GET endpoint can optionally return related resources, if requested with [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources). The following resource types are available:

Reviewer: ```suggestion
To retrieve related resources, you can optionally add [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources) when making `GET` requests. The following resource types are available:
```
